### ISSUE TITLE
Add database in formats supported by LibreOffice/OpenOffice

### ISSUE THREAD

**Issue by ghost** [segor/zno-opendata#1]
**HSQLDB**

* https://en.wikipedia.org/wiki/HSQLDB

**ODF Database**

* https://en.wikipedia.org/wiki/OpenDocument_Format
* https://www.loc.gov/preservation/digital/formats/fdd/fdd000441.shtml

**Comment by segor** [comment]
https://aandreasen.wordpress.com/2010/04/05/connecting-to-ms-sql-server-from-openoffice-base-using-jtds-jdbc-driver-on-linux/  ?

**Comment by ghost** [comment]
> ?

No, it requre "MS SQL server". Its not the same as open database file directly in LO/OO.

**Comment by segor** [comment]
Fill free to port this project to any language or RDBMS
